lstchain/reco/r0_to_dl1.py

```diff
@@ -225,10 +225,10 @@ def r0_to_dl1(
             "allowed_tels": [1],
             "calibrate_flatfields_and_pedestals": False,
             "EventTimeCalculator": {
-                "ucts_t0_dragon": ucts_t0_dragon,
-                "dragon_counter0": dragon_counter0,
-                "ucts_t0_tib": ucts_t0_tib,
-                "tib_counter0": tib_counter0
+                "ucts_t0_dragon": int(ucts_t0_dragon),
+                "dragon_counter0": int(dragon_counter0),
+                "ucts_t0_tib": int(ucts_t0_tib),
+                "tib_counter0": int(tib_counter0)
```

Comment:
No, this is wrong. If it creates an error, it has to be fixed another way!

There cannot be an `int` call here since that will create an error for `None`. It should either be None or already be an integer at this point in the code.

Reply:
> No, this is wrong. If it creates an error, it has to be fixed another way!
> 
> There cannot be an `int` call here since that will create an error for `None`. It should either be None or already be an integer at this point in the code.
> 
> Please revert.

Can't those use -1 as error codes?

Reply:
What error code? These are *options*.

They are the traitlets of the `EventTimeCalculator`

Reply:
I don't see where the problems come from. These come from `argparse` command line options. Just to `type=int` and no default. Then it will be `None` when not given and an `int` when given, exactly as needed.

Reply:
> What error code? These are _options_.
> 
I meant defaults, I thought the problem was using None's as defaults. Perhaps I'm wrong, thoguh.

Reply:
Now I see. You everywhere set non-sensical defaults for the command line options instead of using the fact that argparse can give you `None` for "no option provided".

Reply:
> I meant defaults, I thought the problem was using None's as defaults. 

No, there is not. In fact, it is the "default default".

Reply:
Ok, let's use Nones. So what is the solution for the problem?

Reply:
Removing the `default="NaN"` for the cli option